content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -230,10 +234,11 @@ node {
 <1> Using an inline shell conditional (`sh 'make || true'`) ensures that the
 `sh` step always sees a zero exit code, giving the `junit` step the opportunity
 to capture and process the test reports. Alternative approaches to this are
-covered in more detail in the <<handling-failures>> section below.
+covered in more detail in the <<failure-handling>> section below.
```

Comment:
`failure-handling` an anchor label that I can see.

Reply:
Whoops! Thanks @bitwiseman - this was a remnant of my playing around with gerunds and imperatives for heading titles and I forgot to revert this back when I decided that gerunds were the way forward for heading titles in the User Handbook.